packages/components/src/Toggle/__snapshots__/Toggle.snapshot.test.js.snap

```diff
@@ -8,6 +8,7 @@ exports[`Toggle should render a Checkbox 1`] = `
     htmlFor="id"
   >
     <input
+      autoFocus={undefined}
```

Comment:
What is the benefit ? In DOM using undefine will just not add the attribute. With `false`, we will always have the attribute, most of the time with `false` value.